src/Watchable.cpp

```diff
@@ -43,8 +43,8 @@ std::string Movie::toString() const
 }
 
 Episode::Episode(long id, const std::string& seriesName, int length, int season,
-                 int episode, const std::vector<std::string>& tags)
-: Episode(id, seriesName, length, season, episode, id+1, tags)
+                 int episode, const std::vector<std::string>& tags):
+                 Episode(id, seriesName, length, season, episode, id+1, tags)
```

Comment:
explain this line... somthing here is weird

Reply:
I just moved the `:` to the previous line and indented the constructor. As per the earlier comment suggested

Reply:
ye ye I got it.. i mean explain why Episode constructor is calling to Episode constructor. I know you should. It is just isn't explained...